public/app/core/navigation/parseKeyValue.ts

```diff
@@ -0,0 +1,165 @@
+// tslint:disable
```

Comment:
@dprokop can't see this file used anywhere, do you remember where this was used? maybe something I accidentially removed?

Reply:
Nope, this can be removed. I haves moved the function to utils file and forgot to delete the file